### DISCUSSION TITLE
An option to generate github annotations

### DISCUSSION THREAD

**Discussion by nevinera** [super-linter/super-linter#3598] [Ideas]
It would be incredibly helpful if we could just set an environment variable and have the action write (the first ten) warnings from various linters to stdout in the form of [workflow commands](https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#setting-a-warning-message). It's not there already, as far as I can tell?

**Comment by ccolella-mdc** [comment]
+1. This would be incredibly helpful. It's really really easy for devs to ignore the status result, but really hard to ignore big ugly comments on their changes. That is really all this is missing

**Comment by ferrarimarco** [comment]
This is interesting. Can you please open a new feature request for this?
